lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2363,6 +2375,65 @@ balanceTransactionSpec = describe "balanceTransaction" $ do
     it "produces valid transactions or fails"
         $ property prop_balanceTransactionValid
 
+    describe "bootstrap witnesses" $ do
+        -- Used in 'estimateTxSize' in turn used by coin-selection
+        let coinSelectionEstimatedSize :: Natural -> Natural
+            coinSelectionEstimatedSize
+                = fromIntegral . sizeOf_BootstrapWitnesses . fromIntegral
+
+        let measuredWitSize :: IsCardanoEra era => Cardano.Tx era -> Natural
+            measuredWitSize (Cardano.Tx body wits) = fromIntegral $
+                  serializedSize (Cardano.Tx body wits)
+                - serializedSize (Cardano.Tx body [])
```

Comment:
To avoid non-standard indents, we can use either:

```suggestion
            measuredWitSize (Cardano.Tx body wits) = fromIntegral
                $ serializedSize (Cardano.Tx body wits)
                - serializedSize (Cardano.Tx body [])
```

or

```suggestion
            measuredWitSize (Cardano.Tx body wits) = fromIntegral $
                serializedSize (Cardano.Tx body wits) -
                serializedSize (Cardano.Tx body [])
```